### ISSUE TITLE
Be able to read in facial landmark positions as input from Computer Vision group

### ISSUE THREAD

**Issue by diegocasmo** [diegocasmo/iis-project#14]
- [x] Parse Computer Vision input ([see format](https://gist.github.com/diegocasmo/9f5c025a0202e35f30d5f3c4af6242c7))
- [x] Export/import best model (either MLP or SVM)
- [x] Use best model predict Computer Vision input

Notes:
  - Kaylan mentioned in the meeting that we need to be able to tell what features are the most important ones and communicate that to the Computer Vision group. It would be a good idea to send an email to Kaylan to clarify this requirement.

**Comment by tristaaan** [comment]
So the input is an array of facial expressions?

We can figure out the principal components, but it will vary dataset to dataset.

**Comment by diegocasmo** [comment]
I'm also a bit confused about this, since PCA creates the principal components based on variance, which is dependent on the measurements :/

**Comment by tristaaan** [comment]
I think we have a good start on this in a branch, we need to resolve the accuracy drop (I imagine) we'll get when we take out PCA. Might sit on this a bit for Monday.

**Comment by diegocasmo** [comment]
I may be wrong, but I have been thinking about this, and maybe we won't need to take PCA out of the equation at all, instead we'll used it normally as we always do. PCA selects ``n`` components that have the largest possible variance (not ``n`` features!). So unless the input face is an outlier, this should be fine.

**Comment by tristaaan** [comment]
We need variance across a single column, one row gives no good results. Two rows not much better.

```python
import numpy as np
from sklearn.decomposition import PCA
pca = PCA(n_components = 5, whiten = True)
principal_components = pca.fit_transform(np.array([[1,2,3,4,5,6,7,8,9,11]]))
principal_components
```

**Comment by diegocasmo** [comment]
This should help: [https://github.com/scikit-learn/scikit-learn/blob/a24c8b464d094d2c468a16ea9f8bf8d42d949f84/sklearn/decomposition/base.py#L99](https://github.com/scikit-learn/scikit-learn/blob/a24c8b464d094d2c468a16ea9f8bf8d42d949f84/sklearn/decomposition/base.py#L99). I think ``X`` is ``pca.components_``.

Documentation: [http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html#sklearn.decomposition.PCA.transform](http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html#sklearn.decomposition.PCA.transform)
